### PR TITLE
Use Scalar DB 3.6.0 in Scalar DB sample

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: scalardb-sample/build.gradle [hunk 1]
```diff
@@ -11,7 +11,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.scalar-labs:scalardb:3.5.1'
+    implementation 'com.scalar-labs:scalardb:3.6.0'
```
Comment (brfrn169) [inline scalardb-sample/build.gradle:14]: Use Scalar DB 3.6.0.

File: scalardb-sample/src/main/java/sample/Sample.java [hunk 2]
```diff
@@ -23,8 +21,7 @@ public class Sample implements AutoCloseable {
 
   public Sample() throws IOException {
     // Create a transaction manager object
-    TransactionFactory factory =
-        new TransactionFactory(new DatabaseConfig(new File("database.properties")));
+    TransactionFactory factory = TransactionFactory.create("database.properties");
```
Comment (brfrn169) [inline scalardb-sample/src/main/java/sample/Sample.java:24]: Also, use the new Scalar DB API.

File: scalardb-sample/README.md [hunk 3]
```diff
@@ -118,8 +117,8 @@ Since you use Cassandra in this sample application as mentioned above, you need
 ## Set up
 
 You need to run the following `docker-compose` command:
-```
-docker-compose up -d
+```shell
```
Comment (brfrn169) [inline scalardb-sample/README.md:120]: Also, refactored README a bit.

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by kota2and3kan — APPROVED** [review]
LGTM! Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**